index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <header>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </header>
+<div>
+    <form id="survey-form">
+      <div>
+      <label for="name" id="name-label">Name</label>
+      <input type="text" name="name" id="name">
+      </div>
+
+<div>
+      <label for="email" id="email-label">E-Mail</label>
+      <input type="email" name="email" id="email">
+    </div>
+<div>
+      <label for="number" id="number-label">Age</label>
+      <input type="number" name="age" id="number" min="10" max="100">
+    </div>
+<div>
```

Comment:
Please add two space of indentation on the starting div tags and use maximum of two empty lines. This would improve code readability